Rectangle/mul.lproj/Main.xcstrings

```diff
@@ -2210,7 +2240,7 @@
         "vi" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Spectacle"
+            "value" : "Quanh cảnh"
```

Comment:
Spectacle is the name of an app that came before Rectangle. Honestly, it should be removed from this file, but for this PR you can simply leave it untranslated for Vietnamese.